packages/babel-parser/src/plugins/typescript/index.ts

```diff
@@ -566,6 +566,16 @@ export default (superClass: ClassWithMixin<typeof Parser, IJSXParserMixin>) =>
 
       // For compatibility to estree we cannot call parseLiteral directly here
       node.argument = super.parseExprAtom() as N.StringLiteral;
+      if (this.hasPlugin("importAttributes")) {
+        node.options = null;
+      }
+      if (this.eat(tt.comma)) {
+        this.expectImportAttributesPlugin();
+        if (!this.match(tt.parenR)) {
```

Comment:
Line 569 only checks for the `importAttributes` plugin, but line 573 checks for both. I'm fine with doing either, but we should be consistent.

Reply:
Fixed [c0dafaa](https://github.com/babel/babel/pull/16277/commits/c0dafaaf364dff6844c339a280b821c5abcd8d3a)